sklearn/multioutput.py

```diff
@@ -526,11 +531,20 @@ def _check(self):
 
 
 class _BaseChain(BaseEstimator, metaclass=ABCMeta):
-    def __init__(self, base_estimator, *, order=None, cv=None, random_state=None):
+    @_deprecate_positional_args
```

Comment:
Why using this decorator ? The deprecation for positional args is finished for all estimators, right ?

Reply:
Whoops, mistaken when fixing merge conflicts I think. Will remove